### DISCUSSION TITLE
Fast screenshot superseding slow screenshot

### DISCUSSION THREAD

**Discussion by SlimRunner** [DesModder/DesModder#904] [General]
The update log notes that

> The old approach (slow screenshots) will be removed eventually if there are no problems with "fast screenshots."

However, fast screenshots do not render text. I read the diffs for the feature, so I believe this is a Desmos behavior (I think?). The argument can be made that rendering text in animations is not that common, but it seems troublesome enough to fit within the description of "no problems with fast screenshots".

**Comment by jared-hughes** [comment]
Good catch. The "fast screenshot" is just switching from using `Calc.asyncScreenshot` to `Calc.screenshot`.

Based on the [documentation for Calc.screenshot](https://www.desmos.com/api/v1.9/docs/index.html#GraphingCalculator.screenshot), it doesn't currently support the `showLabels` option. So the "slow screenshots" (asyncScreenshot) cannot be removed until `showLabels` is supported for `Calc.screenshot`.

**Comment by jared-hughes** [comment]
Closing since I found a way to support labels! https://github.com/DesModder/DesModder/pull/915

 After the next release (0.13.9), the "fast screenshot" should be identical, but let me know if there's still a problem.
